### DISCUSSION TITLE
Headless Toggle Component Not Working Properly with Alpine 3.12.0

### DISCUSSION THREAD

**Discussion by kylecotter** [alpinejs/alpine#3453] [7. Bugs (archived)] [answered]
Using the markup provided from the docs - https://alpinejs.dev/component/headless-switch - the toggle does not reflect the `isChecked` background classes on the button despite it being toggled when using Alpine 3.12.0.

Functioning example with Alpine 3.11.1 - https://codepen.io/kylecotter/pen/NWLamGa
Broken example with Alpine 3.12.0 - https://codepen.io/kylecotter/pen/ZEMXZQq

Thanks!

**Comment by SimoTod** [comment]
For some reason, it doesn't look like the ui library was tagged yesterday. Alpine 3.12 will need the latest changes in the ui library to work properly.

**Reply by calebporzio** [reply]
Sorry, I wasn't remembering that I needed to update the UI package this release. Should be all good now!

**Comment by joshhanley** [comment, marked as answer]
@kylecotter you should now be able to update your UI script tag to `<script defer src="https://unpkg.com/@alpinejs/ui@3.12.0-beta.0/dist/cdn.min.js"></script>` to get the latest changes 🙂

**Reply by kylecotter** [reply]
Awesome, all set. Thanks everyone!

**Reply by joshhanley** [reply]
No worries! Thanks for reporting 🙂
